mlir/lib/Dialect/LLVMIR/Transforms/DIScopeForLLVMFuncOp.cpp

```diff
@@ -91,6 +91,10 @@ struct DIScopeForLLVMFuncOp
     Location loc = module.getLoc();
 
     MLIRContext *context = &getContext();
+    if (!context->getLoadedDialect<LLVM::LLVMDialect>()) {
+      emitError(loc, "LLVM dialect is not loaded.");
+      return;
```

Comment:
Maybe also add a `signalPassFailure()` to ensure that a pipeline would stop here.

Reply:
Good idea.